mesheryctl/internal/cli/root/system/model.go

```diff
@@ -413,6 +425,133 @@ mesheryctl exp model search [query-text]
 	},
 }
 
+var importModelCmd = &cobra.Command{
+	Use:   "import",
+	Short: "import model",
+	Long:  "import a model from the registry",
+	Example: `
+
+// Import a model
+mesheryctl exp model import --file <URI>
```

Comment:
If you use angle brackets, this will treat as a HTML tag in respective docs `.md` file
```suggestion
mesheryctl exp model import --file [URI]
```